tracker-pt/module/point_extractor.cpp

```diff
@@ -146,32 +167,38 @@ void PointExtractor::color_to_grayscale(const cv::Mat& frame, cv::Mat1b& output)
     }
     case pt_color_red_chromakey:
     {
-        filter_single_channel(frame, 1, -0.5, -0.5, output);
+        float non_key_coeff = -0.5 * *s.chroma_key_strength;
```

Comment:
You can hoist `non_key_coeff` out of the switch.

Reply:
Okay, will do.